packages/remediations/src/steps/selectPlaybook.js

```diff
@@ -52,115 +61,124 @@ const SelectPlaybook = (props) => {
         fetchData();
     }, []);
 
+    useEffect(() => {
+        if (differenceWith(resolutions, values[RESOLUTIONS], isEqual)?.length > 0) {
```

Comment:
Yup, as I suspected. The `resolutions` selector pulls the data from redux everytime and hence triggering the re-render multiple times. Although this is for triggering new updated to `formOptions` we could be just fine with the `shallowEqual` in `useSelector` and keeping check here as well (we'll shave off a couple re-renders and make this more performant)